NEWS.md

```diff
@@ -21,7 +21,7 @@
   * Added: automatic definition of traits for Active Record enum attributes, enabled by default
     (Note that this required changing where factory_bot constantizes the build
      class, which may affect applications that were using abstract factories for
-     inheritance. See issue #1409.)
+     inheritance. See issue #1409.) (This may break `FactoryBot.lint` because there may be previously non-existing factory+trait combinations being defined and checked)
```

Comment:
This should be wrapped to 80 chars to match the rest of this file.

Reply:
I have amended and push 710dd7adaa520b1c057a93d31b380baea59e2bc6 instead